kolibri/deployment/default/settings/base.py

```diff
@@ -86,6 +87,10 @@
     }
 }
 
+# DIR for storing contentDBs
+CONTENT_DB_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_db"
```

Comment:
I guess what makes me reluctant to use KOLIBRI_HOME is that I need to input this env in the command line every time I run the test. Otherwise things will be happened in the hidden folder outside of my project folder.